templates/block/block--inline-block--banner-deep.html.twig

```diff
@@ -27,6 +27,8 @@
 #}
 {% set classes = ['banner', 'banner--deep'] %}
 <div{{ attributes.addClass(classes) }}>
+  {{ title_prefix }}
```

Comment:
Are these needed if we're not also printing the title variable as well?

Reply:
I think these are needed to get contextual edit links - will double check that and remove them if not the case